docs/querying.md

```diff
@@ -135,6 +135,8 @@ $like: '%hat',         // LIKE '%hat'
 $notLike: '%hat'       // NOT LIKE '%hat'
 $iLike: '%hat'         // ILIKE '%hat' (case insensitive) (PG only)
 $notILike: '%hat'      // NOT ILIKE '%hat'  (PG only)
+$regexp: '^[h|a|t]'    // REGEXP '^[h|a|t]' (MySQL only)
+$notRegexp: '^[h|a|t]' // NOT REGEXP '^[h|a|t]' (MySQL only)
```

Comment:
I was just following the like trend of prepending not. I can remove that if needed. PostgreSQL also allows for CI and CS regexp comparison. Should I add that in (iRegexp, regexp)?